flink-runtime/src/test/java/org/apache/flink/runtime/state/ttl/TtlValueStateTestContext.java

```diff
@@ -21,14 +21,14 @@
 import org.apache.flink.api.common.state.State;
 import org.apache.flink.api.common.state.StateDescriptor;
 import org.apache.flink.api.common.state.ValueStateDescriptor;
-import org.apache.flink.api.common.typeutils.base.StringSerializer;
+import org.apache.flink.api.common.typeutils.base.LongSerializer;
 
 /** Test suite for {@link TtlValueState}. */
 class TtlValueStateTestContext
```

Comment:
Maybe I missed something.
Why do we have to modify this class ?

Reply:
Yes, we need modify this test, since the StringSerializer supports to serialize null value, which hides the original problem.